File: Java/Motor.java
public class Motor {
    private String estado;
    private double potencia;

    public Motor(String estado,double potencia){
        this.estado = estado;
        this.potencia = potencia;
    }

    public void infosMotor(){
        System.out.println("Motor está em "+ this.estado+" estado");
        System.out.println("Potência do motor é de: "+this.potencia);
    }


}

File: Go/Motor.go
package listaRicardo

import "fmt"

type Motor struct {
	Estado   string
	Potencia float64
}

func (m Motor) InfosMotor() {
	fmt.Printf("Motor está em %s estado\n", m.Estado)
	fmt.Printf("Potência do motor é de %.2f cavalos\n", m.Potencia)
}
